datafusion/core/tests/sql/joins.rs

```diff
@@ -3298,3 +3298,105 @@ async fn two_in_subquery_to_join_with_outer_filter() -> Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn right_as_inner_table_nested_loop_join() -> Result<()> {
+    let ctx = create_nested_loop_join_context()?;
+
+    let sql = "SELECT t1.t1_id, t2.t2_id 
```

Comment:
can we add comments to display the distribution required by the join type, like below

```
// Distribution: left is `UnspecifiedDistribution`, right is `SinglePartition`
```